Makefile

```diff
@@ -16,7 +16,7 @@ docker-build: build
 .PHONY: generate
 generate:
 	operator-sdk generate k8s
-	operator-sdk generate crds
+	operator-sdk generate crds --crd-version v1
```

Comment:
v1 should be `v1alpha1` ?

Reply:
No, it should be v1, the first time I adding api are using this version, but I forget why.